src/pages/cli/custom/cdk.mdx

```diff
@@ -18,31 +18,48 @@ amplify add custom
 
 A skeleton CDK stack is generated in the `amplify/backend/custom/<resource-name>` directory. Edit the `cdk-stack.ts` file to define the additional AWS resources. Add additional CDK constructs using the `package.json`. Run `amplify build` to download all NPM dependencies for this custom resource and synthesize the CDK stack.
 
+<Callout>
+
+To migrate from AWS CDK v1 to v2, refer to the [migration documentation](/cli/migration/aws-cdk-migration/).
+
+</Callout>
+
+
 The following example creates an SNS topic whose notification messages are forwarded via email:
 
 ```ts
-import * as cdk from '@aws-cdk/core';
+import * as cdk from "aws-cdk-lib";
 import * as AmplifyHelpers from '@aws-amplify/cli-extensibility-helper';
-import * as sns from '@aws-cdk/aws-sns';
-import * as subs from '@aws-cdk/aws-sns-subscriptions';
+import { AmplifyDependentResourcesAttributes } from "../../types/amplify-dependent-resources-ref";
```

Comment:
we can't import `AmplifyDependentResourcesAttributes` from path . should be from package.

this doesn't seem to be used btw.

Reply:
@sobolk this was auto generated in the cdk file as you can observe this was from path

<img width="737" alt="image" src="https://github.com/aws-amplify/docs/assets/87995712/4871c48b-2c6b-4f3e-861f-58150b773fa1">

Reply:
for reference, we seem to generate that file.
![image](https://github.com/aws-amplify/docs/assets/5849952/33b502bd-40a1-4126-bd3f-629f283f4450)

Reply:
yup we use it here in the code block , will include the import in this PR for documentation purposes
<img width="766" alt="image" src="https://github.com/aws-amplify/docs/assets/87995712/d8038c14-67c5-4ed9-a320-64b3e41d5ba1">